base/actions/actions.py

```diff
@@ -584,10 +625,14 @@ def get_healthcheck_data(self, tracker: Tracker, risk: Text) -> Dict[Text, Any]:
         }
 
         location = self.fix_location_format(tracker.get_slot("location_coords"))
+        city_location = self.fix_location_format(
+            tracker.get_slot("city_location_coords")
+        )
```

Comment:
Should this maybe move into the if below, then we can simplify it quite a bit.

I would think its safe to assume `*_coords` would have values if age != <18

Reply:
I've noticed from several attempts that you do not always get to see the city option especially right after after a previous submission.

Reply:
My comment is more about simplifying the code